install/heroku_postgres/01_deploy_the_collector.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Step 1: Deploy the Collector'
+install_track_title: Installation Guide (Heroku Postgres)
+backlink_href: /docs/install
+backlink_title: 'Installation Guide'
+---
+
+import imgHerokuPaidDyno from "../../images/heroku_paid_dyno.png"
+import { collectorAppName } from "./util";
+
+export const APIKey = ({ apiKey }) => <code>{apiKey}</code>;
+
+export const CollectorApp = ({ organizationSlug }) => (
+  <code>{collectorAppName(organizationSlug)}</code>
+);
+
+In order to monitor a Heroku Postgres database you need to run the pganalyze
+collector as an app inside your Heroku account. Through Heroku's add-on attachment system you then attach all databases that should be monitored.
+
+## Deploy collector
+
+First, deploy the collector to your own Heroku account with your pganalyze API key <APIKey apiKey={props.apiKey} />:
+
+<p>
+  <a
+    href={`https://heroku.com/deploy?template=https://github.com/pganalyze/collector&env[PGA_API_KEY]=${props.apiKey}`}
```

Comment:
It looks like this link somehow doesn't work, since it gets prefixed with the docs page URL:

```
http://localhost:8000/docs/install/heroku_postgres/%7B%60https://heroku.com/deploy?template=https://github.com/pganalyze/collector&env[PGA_API_KEY]=${props.apiKey}`}
```

(this is the actual URL it tried to direct me to)

Reply:
Another MDX rendering weirdness--I worked around with a wrapper component like before. But I also noticed we were setting PGA_API_KEY here even if we didn't have it, so I made that part contingent on having the key.